cmd/app-mesh-inject/main.go

```diff
@@ -65,7 +65,6 @@ func init() {
 	flag.BoolVar(&cfg.InjectXraySidecar, "inject-xray-sidecar", false, "Enable Envoy X-Ray tracing integration and injects xray-daemon as sidecar")
 	flag.BoolVar(&cfg.EnableStatsTags, "enable-stats-tags", false, "Enable Envoy to tag stats")
 	flag.BoolVar(&cfg.EnableStatsD, "enable-statsd", false, "If enabled, Envoy will send DogStatsD metrics to 127.0.0.1:8125")
-	flag.BoolVar(&cfg.InjectStatsDExporterSidecar, "inject-statsd-exporter-sidecar", false, "Deploy statsd_exporter as a sidecar listening on 127.0.0.1:8125 to recieve metrics and export them in prometheus format on 127.0.0.1:9102")
```

Comment:
I think we should leave the flag and just print out a deprecation message, that it doesn't do anything.  That we we don't break anyone for at least a release.

Reply:
Agreed. Have updated the PR.